### ISSUE TITLE
`Box3.expandByObject` not working as expected with 154

### ISSUE THREAD

**Issue by marwie** [mrdoob/three.js#26485]
### Description

`box3.expandByObject` produces wrong/unskinned (?!) bounds for skinnedmesh

### Reproduction steps

1. Call box3.expandByObject with skinnedmesh
2. you can also checkout the two minimal reproduction samples below

tried both with and without computeBoundingBox

### Code

```ts
function fitScene() {
  const size = new Vector3();
  const center = new Vector3();
  const box = new Box3();

  // TODO would be much better to calculate the bounds in camera space instead of world space -
  // we would get proper view-dependant fit.
  // Right now it's independent from where the camera is actually looking from,
  // and thus we're just getting some maximum that will work for sure.

  box.makeEmpty();
  for (const object of scene.children) {
    console.log('EXPAND BY', object);
    box.expandByObject(object, true);
    console.log(box.getSize(new Vector3()));
  }

  camera.updateMatrixWorld();
  camera.updateProjectionMatrix();
  box.getCenter(center);

  // project this box into camera space
  box.applyMatrix4(camera.matrixWorldInverse);

  box.getSize(size);
  box.setFromCenterAndSize(center, size);
  if (size.length() <= 0.0000000001) {
    return;
  }

  const helper = new Box3Helper(box);
  scene.add(helper);
}

```

### Live example

correct bounds with 146: https://stackblitz.com/edit/three-146-expandbyobject-skinnedmesh-works?file=src%2Fmain.ts
incorrect  bounds with 154: https://stackblitz.com/edit/three-154-expandbyobject-skinnedmesh?file=src%2Fmain.ts

### Screenshots

146
![image](https://github.com/mrdoob/three.js/assets/5083203/e2933256-cff7-4631-b830-9b56d09ad9af)

154
![image](https://github.com/mrdoob/three.js/assets/5083203/4b110432-4cc4-496a-b213-63024bd2cfe6)

### Version

154

### Device

Desktop

### Browser

Chrome

### OS

Windows

**Comment by Mugen87** [comment]
There are two issues in your code:

First, the new implementation requires up-to-date matrices. When compute the bounding box in the `onLoad()` event listener, world matrices are not correct. You actually run into #21404. I have tried to fix this with #21411 but unfortunately there was no consensus for a solution.

Besides, using the `precise` parameter with skinned meshes does not provide the correct solution since the method does not honor the bone transformation yet. It might work for certain models but some will break. 

So in your code, do this for now:
```js
scene.updateMatrixWorld(); // FIX 1 
  for (const object of scene.children) {
    console.log('EXPAND BY', object);
    box.expandByObject(object); // FIX 2
    console.log(box.getSize(new Vector3()));
  }
```

**Comment by marwie** [comment]
Thanks for the quick reply @Mugen87 that indeed fixes the issue
